.gitpod.yml

```diff
@@ -1,20 +1,39 @@
-image: ghcr.io/conda/conda-ci:main-linux-python3.9
+image: ghcr.io/maresb/pymc-devcontainer:latest
```

Comment:
can we use a pymc channel here?

Reply:
Yes, but it needs to exist first. To create it, it should be sufficient to merge this, then run the "create devcontainer" GitHub Action (via the workflow dispatch button). Then we can change the channel here.

But this is only needed when changing the dependencies, so for the moment there is no problem to keep my channel.